docs/src/views/Layout.tsx

```diff
@@ -76,6 +76,7 @@ export default () => (
           </OpaqueBackground>
         )
       }}
+      value={`<OpaqueBackground color="lightyellow">This is a bad approach to opaque background :(</OpaqueBackground>`}
```

Comment:
for this component we need to introduce display name as well, will address in a separate PR